src/search.rs

```diff
@@ -161,7 +158,8 @@ where
             } else if child_results.len() == 1 {
                 child_costs.append(&mut child_results);
             } else {
-                todo!("support k > 1");
+                // Pick the lowest cost by O(N) over N implementations.
+                child_costs.push(child_results.into_iter().min().unwrap());
```

Comment:
Would you mind elaborating? I no longer remember what we verified in the last meeting.

Reply:
Sure! The implementation we talked about can be found in the following link. When we recompute the costs for `inner_decisions.len() > 1` in DbBlock::get in db.rs, we choose the lowest action cost since an action is associated with one cost. Similarly, I thought we would want to take the lowest cost when `child_costs.len() > 1` in `top_down_impl` in `search.rs`, but am I wrong?

https://docs.google.com/document/d/1SjUL1Rxd4Kjwa7-4fjXoy2qN6Gafb5_cAsl3gBGfhEs/edit#bookmark=id.34vanum4lehq

Reply:
IIUC, yes. You want to choose the lowest-cost action for each child Spec.

Reply:
I see. Thank you!